### ISSUE TITLE
faux properties in Polymake

### ISSUE THREAD

**Issue by kalmarek** [oscar-system/Polymake.jl#116]
show as capitalized functions in appropriate modules e.g. `$p.DIM` is translated to `Polymake.polytope.DIM(p)`.

should we de-capitalize them??

**Comment by sebasguts** [comment]
I am not sure we should. It is already a mess this way, because some properties are functions. But if we start to just rename them, we might get into even more troubles, as people might not find them in manuals anymore, we need to do it correctly, so we need a list, and so on . . .

**Comment by kalmarek** [comment]
users coming from `polymake` expect this to hide under `p.DIM` so there is no risk of not confusing people;

And here we're attempting to clean-up the mess by not pretending that a function is a property (which is done in polymake only for backwards compatibility (? @benlorenz )

is this the list of faux properties in `pm::polytope`?
```
julia> Polymake.polytope.<TAB>
AMBIENT_DIM
BOUNDED_DUAL_GRAPH
BOUNDED_FACETS
BOUNDED_GRAPH
BOUNDED_HASSE_DIAGRAM
BOUNDED_VERTICES
CD_INDEX
CONNECTIVITY
DIAMETER
DIM
DUAL_CONNECTIVITY
DUAL_DIAMETER
DUAL_EVEN
DUAL_GRAPH_SIGNATURE
DUAL_TRIANGLE_FREE
EHRHART_POLYNOMIAL_COEFF
EVEN
FACET_DEGREES
FACET_POINT_LATTICE_DISTANCES
GALE
GRAPH_SIGNATURE
HILBERT_BASIS
INNER_DESCRIPTION
LATTICE_POINTS
MINKOWSKI_CONE_COEFF
MINKOWSKI_CONE_POINT
N_EDGES
N_FLAGS
N_LATTICE_POINTS_IN_DILATION
N_RIDGES
OUTER_DESCRIPTION
POLYTOPE_IN_STD_BASIS
SCHLEGEL
TRIANGLE_FREE
TRIANGULATION_INT_SIGNS
VERTEX_DEGREES
VERTEX_IN_DEGREES
VERTEX_OUT_DEGREES
VISUAL
VISUAL_BOUNDED_GRAPH
VISUAL_CRUST
VISUAL_DUAL
VISUAL_DUAL_FACE_LATTICE
VISUAL_DUAL_GRAPH
VISUAL_FACE_LATTICE
VISUAL_GRAPH
VISUAL_NN_CRUST
VISUAL_NOP
VISUAL_NOP_GRAPH
VISUAL_ORBIT_COLORED_GRAPH
VISUAL_POINTS
VISUAL_VORONOI
```

**Comment by kalmarek** [comment]
and this is just for discussion to get opinions from you

**Comment by benlorenz** [comment]
Some of these are for backwards compatibility, many are also aliases for convenience. Some of them do pretty-printing, some extract parts of a property to avoid storing it multiple times.

**Comment by sebasguts** [comment]
What really bugs me is that there are fundamental thingies like DIM, and so on. We could do an even more radical solution, btw: Making all the properties seem like methods on the Julia side, and decapitalize them. That way, on the Julia side, there would be no distinction anymore, and nobody can really be confused. The way of calling
`c.FOO` would simply not exist anymore, but everything would be done via `Polytopes.foo(c)`. Of course, this might also lead to troubles

**Comment by kalmarek** [comment]
solved in #119 
('properties', `DIM`, `DEGREE` are available as functions `dim` and `degree` in the appropriate applications)
